packages/server/src/api/controllers/rowAction/crud.ts

```diff
@@ -96,38 +77,36 @@ export async function remove(ctx: Ctx<void, void>) {
   ctx.status = 204
 }
 
-export async function setTablePermission(ctx: Ctx<void, RowActionResponse>) {
+export async function setTablePermission(
+  ctx: Ctx<void, RowActionPermissionsResponse>
+) {
   const table = await getTable(ctx)
   const tableId = table._id!
   const { actionId } = ctx.params
 
   const action = await sdk.rowActions.setTablePermission(tableId, actionId)
   ctx.body = {
-    tableId,
```

Comment:
This data is not actually required, and it would require fetching the automation names. Instead of adding the extra call, we can just simplify the responses